modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -90,13 +90,19 @@ type Handler struct {
 	// to the client immediately.
 	FlushInterval caddy.Duration `json:"flush_interval,omitempty"`
 
+	// A list of IP ranges (supports CIDR notation) from which
+	// X-Forwarded-* header values should be trusted. By default,
+	// no proxies are trusted, so existing values will be ignored
```

Comment:
``?
```suggestion
	// no proxies except those on the local network are trusted,
	// so existing values will be ignored
```

(I'm not sure about this, do you document the Go code or its interaction with the Caddyfile?)